### DISCUSSION TITLE
Version of macOs

### DISCUSSION THREAD

**Discussion by Phoenixjeu** [beeware/briefcase#1837] [General]
Hi to the community
I,m discovering this wonderful tool and am really happy about i will code with it.
i tried the tutorial but when i build the project the .app created is for MacOs11 ..... but i'm with macOS 10.15.7
is there a way to specify that to be able to use the app created ?

thanks a lot

**Comment by freakboy3742** [comment]
No, we don't support macOS 10.15. That release has been EOL and not receiving security updates for almost 2 years.
